packages/asc-ui/src/components/Tabs/Tabs.tsx

```diff
@@ -28,6 +28,10 @@ export interface TabsProps {
    * The children for the tabs, only `Tab` components are allowed.
    */
   children: ReactElement<PropsWithChildren<TabProps>, typeof Tab>[]
+  /**
+   * The index of the initial active tab
+   */
+  initialTabIndex?: number
```

Comment:
Would it not make more sense to use `initialTab` here with the ID of the tab we want to select?